i18n/fr/faq.md

```diff
@@ -68,31 +68,6 @@ Deuxièmement, malgré que l'implémentation en Java soit bonne, elle s'accompag
 | De fournir du financement à des développeurs, hackers et chercheurs via FFS et HackerOne | Le site internet de Kovri ([getkovri.org](getkovri.org) / [kovri.i2p](kovri.i2p)) | D'avoir besoin d'une machine virtuelle Java ultra-performante |
 | Il vise les standards les plus rigoureux en terme de développement et qualité de code | Une documentation ehaustive | De promener votre chien et de payer vos taxes |
 
-- De devenir un nœud sur le réseau I2P
-- De cacher votre localisation physique pour les sites que vous visitez
-- De vous anonymiser sur IRC et vous permet d'envoyer des e-mails de manière anonyme
-- D'héberger des sites et services anonymes
-- De fournir du financement à des développeurs, hackers et chercheurs via FFS et HackerOne
-- Il vise les standards les plus rigoureux en terme de développement et qualité de code
-
-### Qu'est-ce qui est en cours de développement ?
-
-- Émettre des transactions Monero à travers I2P
-- Une GUI pour une utilisation et une configuration améliorées
-- Une bibliothèque d'APIs pour des apps/libs externes
-- Une extension Firefox pour accéder facilement aux *eepsites* (les websites hébergés sur le réseau I2P)
-- Le site internet de Kovri ([getkovri.org](getkovri.org) / [kovri.i2p](kovri.i2p))
-- Une documentation ehaustive
-
-### Qu'est-ce que Kovri ne permettra pas ?
-
-- De sacrifier votre sécurité et vie privée pour des arrière-pensées
-- De fournir une interface web compliquée et pénible
-- D'avoir besoin des autorités pour arriver à un consensus sur le réseau
-- D'accéder à des sites internet via un "outproxy"
-- D'avoir besoin d'une machine virtuelle Java ultra-performante
-- De promener votre chien et de payer vos taxes
-
```

Comment:
This commit may be a conflict of interest between you and @rehrar. I'll leave any final decisions to the both of you.

Reply:
No it is me who proposed to split the question "What does Kovri do/not do for you?" in 3 different questions because the markdown table did not display well on the website. But @rehrar told me he would fix it so I removed my 3 questions and put back the table. Sorry for the confusing commits.

Reply:
Ok, again thanks for clarifying. @rehrar never mentioned this to me.

Reply:
Sorry but I did not see that @rehrar actually also removed the table, splitted in 3 questions and committed. I thought he wanted to keep the table. So you were right with the conflict. I will adapt the french.

Reply:
>Sorry I'm learning the process.

That's ok. I'm in `#monero-translations` now but `#kovri-dev` should always be a primary point of contact for coordination.

Reply:
>#kovri-dev should always be a primary point of contact for coordination.    

ACK, will make sure there is more coordination between the two channels